index.js

```diff
@@ -77,10 +78,13 @@ class TimeoutError extends Error {
 
 const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
 
-const timeout = (promise, ms) => Promise.race([
+const timeout = (promise, ms, abortController) => Promise.race([
```

Comment:
I don't see this parameter being passed anywhere? https://sourcegraph.com/github.com/sindresorhus/ky@8c5e710e8a5adf55985eb1fb2d747ca49204e54e/-/blob/index.js#L81:7&tab=references

Reply:
Unless I'm mistaken, I think it's still not being passed. I don't think this should have been marked resolved.